content/en/docs/collector/internal-telemetry.md

```diff
@@ -57,7 +63,7 @@ service:
   telemetry:
     metrics:
       level: detailed
-```
+
```

Comment:
We'll need these back ticks to close the yaml code block. Could you add them again?